info/v1/machine.go

```diff
@@ -26,6 +26,9 @@ type FsInfo struct {
 
 	// Total number of inodes available on the filesystem.
 	Inodes uint64 `json:"inodes"`
+
+	// HasInodes when true, indicates that Inodes info will be available.
+	HasInodes bool `json:"has_inodes"`
```

Comment:
Sorry, you're right: there's no precedent for doing this with Machine. I added a comment above re: Container FsInfo.